pkgs/development/tools/build-managers/msbuild/nuget.nix

```diff
@@ -0,0 +1,1130 @@
+{ fetchurl }: let
+
+  fetchNuGet = { url, name, version, sha256 }: fetchurl {
+    inherit name url sha256;
+  };
+
+in [
+(fetchNuGet {
```

Comment:
Is this generated?

Reply:
Yes, I adapted what https://github.com/NixOS/nixpkgs/tree/master/pkgs/servers/nosql/eventstore does.